### PR TITLE
Typecheck remaining files in src/api

### PR DESCRIPTION
**Pull request by lumberj**

### CODE REVIEW

File: src/api/common/schema-validator.js [hunk 1]
```diff
@@ -16,9 +18,10 @@ function isValidLedgerHash(ledgerHash) {
   return core.UInt256.is_valid(ledgerHash);
 }
 
-function loadSchema(filepath) {
+function loadSchema(filepath: string) {
   try {
-    return JSON.parse(fs.readFileSync(filepath, 'utf8'));
+    const schemaContent: any = fs.readFileSync(filepath, 'utf8');
```
Comment (lumberj) [inline src/api/common/schema-validator.js]: This is due to this issue, https://github.com/facebook/flow/issues/416

File: src/api/common/utils.js [hunk 2]
```diff
@@ -33,7 +38,10 @@ function wrapCatch(asyncFunction: () => void): () => void {
   };
 }
 
-function composeAsync(wrapper, callback) {
+type Callback = (err: any, data: any) => void
+type Wrapper = (dat: any) => any
```
Comment (clark800) [inline src/api/common/utils.js]: Is this supposed to be `data`?

Comment (lumberj) [inline src/api/common/utils.js]: Yes!

**Comment by clark800** [comment]
LGTM

**Pull request merged**